TLM/TLM/Custom/Data/CustomVehicle.cs

```diff
@@ -28,7 +28,7 @@ public static class CustomVehicle {
 				while (trailingVehicle != 0) {
 					vehManager.m_vehicles.m_buffer[trailingVehicle].Spawn(trailingVehicle);
 					trailingVehicle = vehManager.m_vehicles.m_buffer[trailingVehicle].m_trailingVehicle;
-					if (++numIter > VehicleManager.MAX_VEHICLE_COUNT) {
+					if (++numIter > vehManager.m_vehicles.m_buffer.Length) {
```

Comment:
please use `Constants.ServiceFactory.NetService.MaxVehicleCount` here

Reply:
Maybe better `IVehicleService`? The max vehicle number is something about vehicles, not network segments and nodes.

Reply:
@VictorPhilipp @aubergine10 ping

Reply:
@dymanoid I know I'm not victor or aubergine, but I think that would work better. Quite frankly I'm not sure why we would put it in NetService...

Reply:
Victor knows the mod inside and out, so I'd heed his advice for now.

As part of transition to Harmony (#119) everything is being reorganised so it will likely move to more logical location as part of that.

Reply:
Sorry, my fault - @dymanoid `IVehicleService` is correct! 👍